src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write code here
+function sortList(list) {
+  let myList = list;
+
+  myList = document.querySelector('ul');
```

Comment:
Reassigning `myList` to `document.querySelector('ul')` is incorrect if the intention is to sort the list passed as an argument. This line should be removed or modified to work with the `list` parameter.